authentication/README.md

```diff
@@ -0,0 +1,122 @@
+# Authentication
+
+## Multi Authentication
+
+`MultiAuthentication` is a sample project to demonstrate how to use 
+`Authentication` and `Authorization` features implemented by `.net` to achive
+a mechanism for providing authentication for single endpoint from  multiple 
+schemes.
+
+### Using `Authorize` attribute with multiple scheme values
+
+First option is to set _AuthenticationSchemes_ value of `Authorize` attribute
+and register multiple schemes with `AuthenticationBuilder`.
+
+```csharp
+//Program.cs
+
+builder.Services.
+    .AddAuthentication()
+    .AddScheme<AuthenticationSchemeOptions, HandlerOne>("SchemeOne", options => { })
+    .AddScheme<AuthenticationSchemeOptions, HandlerTwo>("SchemeTwo", options => { });
```

Comment:
this is the abstract naming i'm talking about. and `SchemeOne` is even better than what i've suggested `X`, `Y`

Reply:
changed adding schemes to reflect sample code using "Default","Alternative" and "Anonymous" names